allennlp/data/token_indexers/single_id_token_indexer.py

```diff
@@ -29,14 +29,19 @@ def __init__(self, namespace: str = 'tokens', lowercase_tokens: bool = False) ->
 
     @overrides
     def count_vocab_items(self, token: str, counter: Dict[str, Dict[str, int]]):
```

Comment:
Can you change the type annotations here or/and add a comment describing in what case the `SingleIdTokenIndexer` receives ints.